packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -861,6 +853,12 @@ class TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
       ignorePointer: widget.ignorePointer ?? widget.message != null,
     );
 
+    overlayChild = DefaultTextStyle.merge(
```

Comment:
Nit: Could you do this in one line with the declaration of `overlayChild` so that it could be `final`?